Fixed Icons, license, and .Net versions supported

darcythomas opened the pull request.

QuantumNightmare commented on Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj:
```diff
@@ -9,25 +9,36 @@
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```
Just wondering why this dependency was added

darcythomas replied:
I guess I mis-read this. https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#when-to-target-net50-or-net60-vs-netstandard.

Re reading I guess targeting 2.0 and .net5-6 would be best

darcythomas replied:
Hmmm. But this seems to say https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#net-standard-not-deprecated both would be the way to go

QuantumNightmare replied:
Yeah, should be fine to target 2.1, was just wondering why the Abstractions package was also added. Perhaps the IDE added this for you.

QuantumNightmare reviewed (approved):
Looks good, just added one question for my understanding. Also, we didn't want to use a colored icon? (A colored icon appears on the NuGet site, but this monochromatic one appears in Visual Studio. Can barely see it when using the VS dark theme).

darcythomas commented:
> Looks good, just added one question for my understanding. Also, we didn't want to use a colored icon? (A colored icon appears on the NuGet site, but this monochromatic one appears in Visual Studio. Can barely see it when using the VS dark theme).


I just took the icon which was in the iconlink, downloaded and added that.

QuantumNightmare commented:
Not sure where the colored one in the NuGet site is coming from. Just thinking while you're in this space, could optionally confirm with a designer which icon we want to use.

darcythomas commented:
Updated Icon (correct color) and removed un-need package imports

QuantumNightmare reviewed (approved):
Thanks for the changes :+1:

darcythomas commented:
Tested against targets:

<html>
<body>
<!--StartFragment-->

.NET and .NET Core | 1.0, 1.1, 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0
-- | --
.NET Framework  | 4.6.1 2, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8

<!--EndFragment-->
</body>
</html>
